satellite/marketing/marketingweb/static/css/style.css

```diff
@@ -0,0 +1,41 @@
+html{
+    height:100vh;
+    width:100vw;
+}
+
+body{
+    height:100%;
+    width:100%;
+    background-color: #DDDDDD;
+}
+
+.home-link:hover,.home-link p:hover{
+    text-decoration: none;
+}
+
+.banner{
+    height: 80px;
+    overflow: hidden;
+}
+
+.banner-txt{
+    font-family: 'Inter';
+    font-size: 18px;
+    color:white;
+}
+
+.offer-label{
+font-family: 'Inter';
```

Comment:
nit pick: the indentation here is a little off

Reply:
done